### DISCUSSION TITLE
Localization support

### DISCUSSION THREAD

**Discussion by loranger** [laravel-zero/laravel-zero#373] [Ideas]
Hello,

Maybe it would be convenient to support localization ?
I use to translated my strings in my localized app, but I just had to set locale on Carbon, and thought it would be nice to have full translated apps, with localized dates, prompts and so on…

**Comment by owenvoke** [comment]
I think you can probably require `illuminate/translation` and then register `Illuminate\Translation\TranslationServiceProvider` in your `config/app.php`.

You should then be able to use `trans('your_key')` as with Laravel afaik.

I guess it might be quite good to have a component for this. 🤷🏻
